thoth/common/openshift.py

```diff
@@ -1074,17 +1075,14 @@ def generate_id(
         prefix: Optional[str] = None, identifier: Optional[str] = None
     ) -> str:
         """Generate an identifier."""
-        # A very first method used 'generatedName' in ImageStream configuration,
-        # but it looks like there is a bug in OpenShift as it did not generated any
-        # name and failed with regexp issues (that were not related to the
-        # generateName configuration).
+        suffix = f"-{datetime.datetime.now():%y%m%d%H%M%S}-{random.getrandbits(64):08x}"
```

Comment:
No "-" required at the beginning of the format string as it is added later.

Reply:
Thanks for the review, nice catch :+1:

Added at the end to keep function call compatible with the previous implementation. Thanks!